src/lightning_app/core/app.py

```diff
@@ -45,6 +46,7 @@ class LightningApp:
     def __init__(
         self,
         root: "lightning_app.LightningFlow",
+        flow_cloud_compute: Optional["lightning_app.CloudCompute"] = None,
```

Comment:
Let's call it `default_cloud_compute`. I think it is more accurate.

Reply:
This was also my suggestion above, but the logic that sets this as default also on the Work is missing. It would have to be done either here or in a different PR. But we are close to a release, so this should be considered carefully I think.

Reply:
this change was reverted because it lead to more discussions

Reply:
I think this requires a final API decision. Flow doesn't seem right as it runs the flow, rest api, frontends and in the future the works.
I agree default is mis-leading and I don't think it is a good candidate.

I think `main_cloud_compute` or `root_cloud_compute` would fit better.